apps/spotlight/src/pages/splash/index.astro

```diff
@@ -4,67 +4,111 @@ import atjInfographics from '../../images/atj-infographics.png';
-      mozallowfullscreen
-      allowfullscreen
-      style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"
-    ></iframe>
+  <div class="offline-homepage">
+    <section class="usa-hero" aria-label="Introduction">
+      <div class="grid-container">
+        <div class="usa-hero__callout padding-0">
+          <h1 class="usa-hero__heading font-sans- text-base-darkest">
+            A people-centered platform for online document assembly
+          </h1>
+          <p class="play-video margin-top-0">
+            <a href="#demo-video"
+              class="usa-button text-base-lightest bg-base-darkest" 
+              aria-controls="demo-video"
+              data-open-modal     
+            > 
+              <span class="font-sans-3xs text-normal display-block margin-bottom-1">Play Video | 5 minutes</span>
+              <span class="sans-body-xs display-block">Demo: 10x Access to Justice Forms Platform</span>
+            </a>
+          </p>         
+        </div>
+        <div
+          class="usa-modal usa-modal--lg"
+          id="demo-video"
+          aria-labelledby="modal-1-heading"
+          aria-describedby="modal-1-description"
+        >
+          <div class="usa-modal__content">
+            <div class="usa-modal__main">
+              <div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/d958328ec69548ed99c6cd1a15af8a16?sid=ecc5f80b-931b-403d-addc-538215b53092" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+            </div>
+            <button
+              type="button"
+              class="usa-button usa-modal__close"
+              aria-label="Close this window"
+              data-close-modal
+            >
+              &#x2715;
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="usa-section">
+        <div class="grid-container intro-block">
+          <p class="usa-intro font-sans-9 text-light margin-0">The 10x Access to Justice Forms Platform is revolutionizing the way courts and government agencies handle forms, replacing outdated fillable PDFs with modern, web-based digital forms that are compliant with the 
+            <a
+              class="usa-link usa-link--external"
+              rel="noreferrer"
+              href="https://digital.gov/resources/delivering-digital-first-public-experience/"
+              >
+                21st Century Integrated Digital Experience Act (IDEA)
+              </a>
+          </p>
+        </div>
+      </section>
+      <section class="usa-graphic-list usa-section bg-base-darkest text-white">
+          <div class="grid-container">
+            <div class="usa-graphic-list__row grid-row grid-gap">
+              <p>
+                <img src={atjInfographics.src} role="img" alt="10x ATJ Infographic" />
```

Comment:
This infographic has a lot of text. If it's possible to capture a synopsis of what the text says, that would be a good addition here. Also, this image is below the fold. If possible, it would be good to put `height`, `width`, and `loading="lazy"` attributes on them.

Reply:
Done.